test/hack/cleanup/main.go

```diff
@@ -93,3 +105,26 @@ func main() {
 		}
 	}
 }
+
+func cleanUpCluster(ctx context.Context, name string, client *eks.Client, log *zap.SugaredLogger, expirationTime time.Time) {
+	if name != "" {
+		if strings.HasPrefix(name, "soak-") {
+			cluster, err := client.DescribeCluster(ctx, &eks.DescribeClusterInput{Name: aws.String(name)})
+			if err != nil {
+				log.Errorf("%v", err)
+			}
+
+			if !lo.FromPtr(cluster.Cluster.CreatedAt).Before(expirationTime) {
+				log.Infof("soak testing cluster (%s) will not need to be cleanup", name)
```

Comment:
Can you also add some info on when the cluster _should_ be cleaned up?
```suggestion
				log.Infof("soak testing cluster (%s) does not need to be cleaned up", name)
```